types/svelte-leafletjs/index.d.ts

```diff
@@ -0,0 +1,211 @@
+// Type definitions for svelte-leafletjs 0.8
+// Project: https://github.com/ngyewch/svelte-leaflet
+// Definitions by: Kyle Hensel <https://github.com/k-yle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.2
```

Comment:
```suggestion
```
why do you need 4.2 version?

Reply:
It's required to use [template literal types here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7ba9e0/types/svelte-leafletjs/util.d.ts#L2). This reduces the maintenance burden, otherwise we would have to copy-paste the types from the other DT package and keep them in sync.

Reply:
thx!